src/main/scala/ohnosequences/loquat/utils.scala

```diff
@@ -23,28 +25,21 @@ case object utils {
     }
   }
 
-
-  class Time(val inSeconds: Long) {
-    val millis: Long = inSeconds * 1000
-    val seconds: Long = inSeconds
-    val minutes: Long = inSeconds / 60
-    val hours: Long = inSeconds / (60 * 60)
-
-    def prettyPrint: String = List(
-      (hours, "hours"),
-      (minutes, "min"),
-      (seconds, "sec")
-    ).map{ case (value, label) =>
-      (if (value > 0) s"${value} ${label}" else "")
-    }.mkString
+  // A minimal wrapped around the Java scheduling thing
+  // Note, that this ScheduledFuture has cancel(Boolean) method
+  def schedule(
+    after: FiniteDuration,
+    every: FiniteDuration
+  )(block: => Unit): ScheduledFuture[_] = {
+
+    new ScheduledThreadPoolExecutor(1).scheduleAtFixedRate(
+      new Runnable { def run(): Unit = block },
+      after.toSeconds,
+      every.toSeconds,
+      SECONDS
+    )
```

Comment:
@eparejatobes could you take a look at this thing? is it ok? I'm not sure about this pool executor thing with `corePoolSize = 1`.

In the end I used Java [`ScheduledExecutorService`](http://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ScheduledExecutorService.html) thing instead of introducing dependency on akka.

This simple wrapper will be used for three things (to avoid manual Threads launching with `while(true)` and `Thread.sleep`):
- log uploading
- checking self-termination conditions
- extending SQS message visibility timeout while the task is processed

The rest can be done with using scala Future.